app/components/footer-main.js

```diff
@@ -2,13 +2,13 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 
 export default Component.extend({
-  tagName    : 'footer',
-  classNames : ['ui', 'inverted', 'vertical', 'footer', 'segment'],
-
-  currentLocale: computed(function() {
+  tagName       : 'footer',
+  classNames    : ['ui', 'inverted', 'vertical', 'footer', 'segment'],
+  events        : ['Appearance or Signing', 'Attraction', 'Camp, Trip, or Retreat', 'Class, Training, or Workshop', 'Concert or Performance', 'Conference', 'Convention', 'Dinner or Gala', 'Festival or Fair', 'Game or Competition'],
```

Comment:
@Kreijstal this is not what I meant. These should be coming in from the server. FE does not control any of this.

Reply:
Are these already defined from the server?